python-main.js

```diff
@@ -627,11 +627,22 @@ function web_editor(config) {
         $("#command-share").click(function () {
             doShare();
         });
+
+        function formatHelpPanel(){
+            if($(".helpsupport_container").offset().left !== $("#command-help").offset().left && $(window).width() > 620){
+                console.log("wow");
+                $(".helpsupport_container").css("top", $("#command-help").offset().top + $("#toolbox").height() + 10);
+                $(".helpsupport_container").css("left", $("#command-help").offset().left);
+        }
```

Comment:
Just a nitpick, but since you have to remove the console.log, you can also fix the indentation here.